Makefile

```diff
@@ -1,8 +1,32 @@
 SNAPPY = $(CHAIN)/vendor/github.com/google/snappy
 ROCKSDB = $(CHAIN)/vendor/github.com/facebook/rocksdb
+DB_DEV = core
+RAFT_DEV = $(HOME)/.chaincore/raft
```

Comment:
This will only work if CHAIN_CORE_HOME is unset or matches ~/.chaincore. (See https://github.com/chain/chain/blob/main/core/env.go)

I am not too worried about this kind of thing, but can we add a comment at the top of this file explaining that this is for build/development purposes and/or may not work in all environments, or something like that?